play-json/shared/src/main/scala-2.13-/play/api/libs/json/ImmutableLinkedHashMap.scala

```diff
@@ -20,8 +20,9 @@ import scala.collection.mutable.ArrayBuffer
 private[json] class ImmutableLinkedHashMap[A, +B](underlying: JLinkedHashMap[A, B])
     extends AbstractMap[A, B]
     with Map[A, B]
-    with MapLike[A, B, ImmutableLinkedHashMap[A, B]] {
+    with MapLike[A, B, ImmutableLinkedHashMap[A, B]] with Serializable {
```

Comment:
If you look at std lib collections you will notice the use of `scala.collection.generic.DefaultSerializable`

Reply:
DefaultSerializable available only with Scala2.13 and serialization tests failing with ClassCastException then..